enhancements/ptp-time-integration.md

```diff
@@ -165,6 +160,13 @@ This proposal doesn't support timemaster service configuration. Use of PTP will
 require disabling NTP(Chrony) first. No redundancy is provided between NTP and
 PTP in this proposal.
 
+##### PTP on multiple ports
+
+Ptp4l supports to be run multiple times on different interfaces.(e.g. using timemaster).
```

Comment:
Do we need multiple ptp domains with timemaster or ptp on bond for redundancy?

Reply:
This PR is about adding mutiple domains